tests/runtime/dwarf

```diff
@@ -45,6 +45,62 @@ REQUIRES_FEATURE dwarf
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
+NAME uprobe skip inlined function
+PROG config = { probe_inline = 0 }
+     uprobe:./testprogs/inline_function:square { @count++ }
+     uretprobe:./testprogs/inline_function:main { exit() }
+EXPECT @count: 1
```

Comment:
I guess bpftrace initializes this map value to 1 even though it never gets called?

Reply:
No, it should be initialized to 0. The test checks that only one (non-inlined) instance of `square` is hit (see the `inline_function.c` source).